src/Output/Output_DumpData_Part.cpp

```diff
@@ -194,7 +197,23 @@ void Output_DumpData_Part( const OptOutputPart_t Part, const bool BaseOnly, cons
                   EdgeL  = amr->patch[0][lv][PID]->EdgeL;
                   EdgeR  = amr->patch[0][lv][PID]->EdgeR;
 
-                  if ( Part == OUTPUT_DIAG ) // (+1,+1,+1) diagonal
+                  if ( Part == OUTPUT_BOX ) // entire box
+                  {
+//                   compute the derived fields
+                     GetDerivedField( Der_FluIn, Der_Out, Der_MagFC, Der_MagCC, lv, PID, Der_PrepFluIn );
+                     Der_PrepFluIn = false;
+
+//                   write data
+                     for (int k=0; k<PS1; k++)  {  kk = Corner[2] + k*scale;
+                     for (int j=0; j<PS1; j++)  {  jj = Corner[1] + j*scale;
+                     for (int i=0; i<PS1; i++)  {  ii = Corner[0] + i*scale;
+
+                        WriteFile( File, lv, PID, i, j, k, ii, jj, kk, Der_Out );
+
+                     }}} // i, j, k
+                  } // if ( Part == OUTPUT_BOX )
```

Comment:
This code snippet seems unnecessary if you set `Check_x = Check_y = Check_z = false` for `OUTPUT_BOX`, allowing it to be treated the same as, for example, `OUTPUT_X`. How about removing it to reduce redundant code?